certora/specs/StayHealthy.spec

```diff
@@ -25,8 +25,12 @@ filtered {
     // Safe require because of the invariant sumBorrowSharesCorrect.
     require borrowShares(id, user) <= totalBorrowShares(id);
 
-    bool stillHealthy = isHealthy(marketParams, user);
-    assert !priceChanged => stillHealthy;
+    assert isHealthy(marketParams, user);
+}
+
+function mulDivUp(uint256 x, uint256 y, uint256 d) returns uint256 {
+    assert d != 0;
+    return assert_uint256((x * y + (d - 1)) / d);
```

Comment:
Note that this is not making any assumption as `summaryMulDiv*` were doing before